python/tests/test_genotype_matching_fb.py

```diff
@@ -737,29 +760,35 @@ def compute_next_probability_dict(
         is_match,
         template_is_het,
         query_is_het,
+        query_is_missing,
     ):
         mu = self.mu[site_id]
 
         template_is_hom = np.logical_not(template_is_het)
-        query_is_hom = np.logical_not(query_is_het)
 
-        EQUAL_BOTH_HOM = np.logical_and(
-            np.logical_and(is_match, template_is_hom), query_is_hom
-        )
-        UNEQUAL_BOTH_HOM = np.logical_and(
-            np.logical_and(np.logical_not(is_match), template_is_hom), query_is_hom
-        )
-        BOTH_HET = np.logical_and(template_is_het, query_is_het)
-        REF_HOM_OBS_HET = np.logical_and(template_is_hom, query_is_het)
-        REF_HET_OBS_HOM = np.logical_and(template_is_het, query_is_hom)
-
-        p_e = (
-            EQUAL_BOTH_HOM * (1 - mu) ** 2
-            + UNEQUAL_BOTH_HOM * (mu**2)
-            + REF_HOM_OBS_HET * (2 * mu * (1 - mu))
-            + REF_HET_OBS_HOM * (mu * (1 - mu))
-            + BOTH_HET * ((1 - mu) ** 2 + mu**2)
-        )
+        if query_is_missing:
+            p_e = 1
+        else:
+            query_is_hom = np.logical_not(query_is_het)
+
+            EQUAL_BOTH_HOM = np.logical_and(
```

Comment:
All caps like this is usually reserved for constants (https://peps.python.org/pep-0008/#constants) is there a convention here that I'm not seeing?